### PR TITLE
Fix AD queries

### PR DESCRIPTION
**Pull request by yaakovi**
fixes: https://github.com/demisto/etc/issues/11030

### CODE REVIEW

File: Scripts/script-ADGetUser.yml [hunk 1]
```diff
@@ -256,7 +264,6 @@ args:
   description: mail attribute of desire user
 - name: userAccountControlOut
   auto: PREDEFINED
-  description: Get translated user acount control flags
```
Comment (dorsha) [inline Scripts/script-ADGetUser.yml]: Why did you remove the description?

File: Scripts/script-ADGetUser.yml [hunk 2]
```diff
@@ -302,6 +309,8 @@ outputs:
   description: The indicator's score
   type: number
 scripttarget: 0
+runonce: false
 dependson:
   must:
   - ad-search
+releaseNotes: "add escaping to filter"
```
Comment (dorsha) [inline Scripts/script-ADGetUser.yml]: Release notes should start with capital and should be explain better the fix (users will not understand this..)

File: Scripts/script-ADGetUser.yml [hunk 3]
```diff
@@ -32,6 +32,13 @@ script: |-
   if demisto.get(demisto.args(), 'limit'):
       limit = demisto.args()['limit']
 
+
+  def escapeSpecialCharacters(s):
+      for c in r'\()':
```
[the remark this answers is not in the document]
Comment (dorsha) [reply Scripts/script-ADGetUser.yml]: why not using regex group replace?

File: Scripts/script-ADGetUser.yml (hunk 1 above)
Comment (yaakovi) [reply Scripts/script-ADGetUser.yml]: there were two description lines, when downloading the integration it removed the first automatically.

Comment (dorsha) [reply Scripts/script-ADGetUser.yml]: 👍

File: Scripts/script-ADGetUser.yml [hunk 4]
```diff
@@ -32,6 +34,11 @@ script: |-
   if demisto.get(demisto.args(), 'limit'):
       limit = demisto.args()['limit']
 
+
+  def escapeSpecialCharacters(s):
```
Comment (asafshen) [inline Scripts/script-ADGetUser.yml]: what happens if there are two agencent brackets?
image
  
it seems that only the second `(` is missing escape characters..isn't it?

**Comment by anara123** [comment]
@asafshen @yaakovi what is status here?
